tools/bin/ci_credentials.sh

```diff
@@ -1,26 +1,150 @@
 #!/usr/bin/env bash
 
 . tools/lib/lib.sh
+. tools/lib/gcp-token.sh
 
 set -e
 
+COMMAND_NAME=$1
+
+# all secrets will be loaded if the second argument is not present
+CONNECTOR_FULLNAME=${2:-all}
+CONNECTOR_NAME=`echo ${CONNECTOR_FULLNAME} | rev | cut -d'/' -f1 | rev`
+
+GSM_SCOPES="https://www.googleapis.com/auth/cloud-platform"
+
+declare -A SECRET_MAP
+
+
 function write_standard_creds() {
   local connector_name=$1
   local creds=$2
   local cred_filename=${3:-config.json}
+  local source_name=${4:-github}
```

Comment:
I suggest renaming this to `secrets_provider_name` to avoid overloading with connector sources